event_samples/eventbridge-sqs.json

```diff
@@ -0,0 +1 @@
+{"Records":[{"messageId":"e995e54f-1724-41fa-82c0-8b81821f854e","receiptHandle":"AQEB4mIfRcyqtzn1X5Ss+ConhTejVGc+qnAcmu3/Z9ZvbNkaPcpuDLX/bzvPD/ZkAXJUXZcemGSJmd7L3snZHKMP2Ck8runZiyl4mubiLb444pZvdiNPuGRJ6a3FvgS/GQPzho/9nNMyOi66m8Viwh70v4EUCPGO4JmD3TTDAUrrcAnqU4WSObjfC/NAp9bI6wH2CEyAYEfex6Nxplbl/jBf9ZUG0I3m3vQd0Q4l4gd4jIR4oxQUglU2Tldl4Kx5fMUAhTRLAENri6HsY81avBkKd9FAuxONlsITB5uj02kOkvLlRGEcalqsKyPJ7AFaDLrOLaL3U+yReroPEJ5R5nwhLOEbeN5HROlZRXeaAwZOIN8BjqdeooYTIOrtvMEVb7a6OPLMdH1XB+ddevtKAH8K9Tm2ZjpaA7dtBGh1zFVHzBk=","body":"{\\"version\\":\\"0\\",\\"id\\":\\"af718b2a-b987-e8c0-7a2b-a188fad2661a\\",\\"detail-type\\":\\"my.Detail\\",\\"source\\":\\"my.Source\\",\\"account\\":\\"425362996713\\",\\"time\\":\\"2023-08-03T22:49:03Z\\",\\"region\\":\\"us-east-1\\",\\"resources\\":[],\\"detail\\":{\\"text\\":\\"Hello, world!\\",\\"_datadog\\":{\\"x-datadog-trace-id\\":\\"7379586022458917877\\",\\"x-datadog-parent-id\\":\\"2644033662113726488\\",\\"x-datadog-sampling-priority\\":\\"1\\",\\"x-datadog-tags\\":\\"_dd.p.dm=-0\\",\\"traceparent\\":\\"00-000000000000000066698e63821a03f5-24b17e9b6476c018-01\\",\\"tracestate\\":\\"dd=t.dm:-0;s:1\\"}}}","attributes":{"ApproximateReceiveCount":"1","AWSTraceHeader":"Root=1-64cc2edd-112fbf1701d1355973a11d57;Parent=7d5a9776024b2d42;Sampled=0","SentTimestamp":"1691102943638","SenderId":"AIDAJXNJGGKNS7OSV23OI","ApproximateFirstReceiveTimestamp":"1691102943647"},"messageAttributes":{},"md5OfBody":"93d9f0cd8886d1e000a1a0b7007bffc4","eventSource":"aws:sqs","eventSourceARN":"arn:aws:sqs:us-east-1:425362996713:lambda-eb-sqs-lambda-dev-demo-queue","awsRegion":"us-east-1"}]}
```

Comment:
TODO - format

Reply:
This Payload doesn't seem to be EventBridge-SQS, unless I'm missing something, but I don't see any `Records` on your `body`.

```json
"body": {
    "version": "0",
    "id": "af718b2a-b987-e8c0-7a2b-a188fad2661a",
    "detail-type": "my.Detail",
    "source": "my.Source",
    "account": "425362996713",
    "time": "2023-08-03T22: 49: 03Z",
    "region": "us-east-1",
    "resources": [],
    "detail": {
        "text": "Hello, world!",
        "_datadog": {
        "x-datadog-trace-id": "7379586022458917877",
        "x-datadog-parent-id": "2644033662113726488",
        "x-datadog-sampling-priority": "1",
        "x-datadog-tags": "_dd.p.dm=-0",
        "traceparent": "00-000000000000000066698e63821a03f5-24b17e9b6476c018-01",
        "tracestate": "dd=t.dm: -0;s: 1"
        }
    }
},
```

Reply:
One must reverse their brain :D

This is a message received by a Lambda function, which was from an SQS queue, and before that, an Event Bus.

So `Records`, which is the first key of this JSON object, represents all the records of the SQS payload. The first item is a message, and that first message has a body, which is the EventBridge Event.

Reply:
Oh lol, I totally missed that, thank you.

Reply:
No worries, I have to cross my eyes to see it!